src/components/yeti/map-layers/YetiLayer.vue

```diff
@@ -164,6 +162,7 @@ export default {
       this.mapLegend.items.forEach((item) => {
         item.color = `rgb(${item.color[0]}, ${item.color[1]}, ${item.color[2]})`;
       });
+      this.layer.getSource().setAttributions('<li>| ' + this.mapLegend.attributions + '</li>');
```

Comment:
is the pipe needed? to me, it should rather be a separator. note that there have been some fixes to attributions (but not in yeti map) in recent commits (your branch is outadated)

Reply:
Well, I used pipe to visually separate each attributions (as it is for default ones). Is the commit (878afbd46707494) you're talking about? Not sure if already deployed, but CSS is not applying...

Reply:
OK, I see. Updated code accordingly.